project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module

```diff
@@ -315,10 +315,13 @@ function c4m_user_profile_form_user_register_form_alter(
   );
   $form['#submit'][] = 'c4m_user_profile_form_user_register_form_submit';
 
-  // Hide status fields - SA-created user is always active.
-  $form['account']['status']['#access'] = FALSE;
-  // Alter 'notify user' checkbox title.
-  $form['account']['notify']['#title'] = t('Send password reset link');
+  // If user is administrator.
+  if (form['administer_users']['#value']) {
```

Comment:
@anvmn 
Please use `user_access('administer users')` as we usually do.
this function is cached so you don't need to worry about performance.